src/hw_isolation_record/manager.cpp

```diff
@@ -652,6 +672,13 @@ void Manager::updateEntryForRecord(const openpower_guard::GuardRecord& record,
         updated = true;
     }
 
+    // TODO:https://github.com/ibm-openbmc/openpower-hw-isolation/issues/39
+    // During "core" checkstop PLDM will be blocked on the DMA transfer of
+    // the dump data and might not honor enabling resolved D-Bus property
+	// of core D-Bus object hosted by PLDM during isolation entry creation.
```

Comment:
Please align

Reply:
it is already aligned, can you please explain what is the issue